VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -367,4 +375,73 @@ public void orientationChangeTest() {
                 activity.getResources().getConfiguration().orientation);
         assertOrientationSetup(activity);
     }
+
+    /**
+     * Testing creation of various fragments in the {@link CourseUnitNavigationActivity}'s
+     * ViewPager, by supplying its {@link CourseUnitPagerAdapter} with all possible
+     * {@link CourseComponent} models.
+     */
+    @Test
+    public void testUnitFragmentCreation() {
```

Comment:
But I just realized that we're using Mockito instead of Robolectric here, so I guess this proposal can work too.